.github/workflows/main.yml

```diff
@@ -61,58 +61,98 @@ jobs:
           docker build --file ./microservices/embeddingservice/docker/Dockerfile ./microservices/embeddingservice
           echo "embedding service image build OK"
 
-  
-  docker_build_publish_dev:
+
+  build_publish_deploy_ingest_dev:
     needs: unit_tests
-    name: docker_build_publish_dev
-    runs-on: ubuntu-latest
+    uses: ./.github/workflows/build-publish-deploy.yml
     if: github.event_name == 'push' && github.ref == 'refs/heads/dev'
+    with:
+      service-name: ingest
+      stage: dev
+    secrets:
```

Comment:
using `secrets: inherit` would be less verbose. See: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-jobsjob_idsecretsinherit
Any reason we want to pass secrets explicitly?